molecule/state.py

```diff
@@ -79,6 +80,10 @@ def multiple_platforms(self):
     def customconf(self):
         return self._data.get('customconf')
 
+    @property
+    def installed_deps(self):
+        return self._data.get('installed_deps')
```

Comment:
Can we get a simple test for this in the state file.  Can just copy an [existing](https://github.com/metacloud/molecule/blob/master/tests/unit/test_state.py#L51-Lundefined) one.

We don't need to really assert it changes, just making sure it doesn't throw an exception due to the property not being there.